integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/FancyFillTest.kt

```diff
@@ -33,3 +36,10 @@ interface FancyFillTest {
 fun FancyFillTest() {
     FancyFillTestDoc.MainFrame(onTap = { Log.e("onTap", "frame clicked!") })
 }
+
+@Composable
+fun SquooshFancyFillTest() {
```

Comment:
Is this referenced from the AllRenderTests file? Ok to do so later -- I think we should probably add a "Squoosh switch" to the validation app so we don't have to write these anymore... (Maybe doing this would be better than updating AllRenderTests...)

Reply:
Filed ticket #1384 to track this.

Right now I am just changing it manually from FancyFillTest to SquooshFancyFillTest in the AllExamples file and rebuild to test.